### PR TITLE
FIX error on click personal setting #459

### PR DESCRIPTION
**Pull request by dalacost**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://dalacost-patch-3.fatfreecrm-fat-free-crm-1bbda9a74e1b4e1e2ffd.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://dalacost-patch-3.fatfreecrm-fat-free-crm-1bbda9a74e1b4e1e2ffd.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: app/views/users/_user.html.haml [hunk 1]
```diff
@@ -15,7 +15,7 @@
         = h(@user.company) unless @user.company.blank?
       %br
       %div
-        #{t :email}: #{auto_link(@user.email)}
+        #{t :email}: #{auto_link(@user.alt_email)}
```
Comment (steveyken) [inline app/views/users/_user.html.haml]: Hi there, I'm not sure this is intended behaviour. We want to show the `email` and optionally the `alt_email`. Is there a problem you were trying to fix? Thanks.

Comment (dalacost) [inline app/views/users/_user.html.haml]: this change solve the issue #459

**Comment by steveyken** [comment]
We'll address this in issue #517

**Pull request closed without merging**